esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -609,7 +609,7 @@ def teachersbyFOO(self, request, tl, one, two, module, extra, prog, sort_exp = l
                 classes = new_classes
 
             # aseering 9-29-2007, 1:30am: There must be a better way to do this...
```

Comment:
This comment can probably be deleted since you've changed the code (hopefully for the better)?